### PR TITLE
Merged multi lists and single lists in a single component

### PR DESCRIPTION
**Pull request by sansyrox**
## Summary

Restyled flat lists and merged multi-column flat lists into a single component with simple flat list.

The Multi List Component was scrapped and merged to only the Flat List Component and simple Lipsum was replaced by meaningful text.

## Changelog

[General] [Changed] - Merged Multilist in a single Flat List and updated FlatList Screens

### Before this PR

<table>

  <tr>
    <td>Screenshot 2020-09-10 at 5 17 48 AM</td>
    <td>Screenshot 2020-09-10 at 5 27 37 AM</td>
    <td>Screenshot 2020-09-10 at 5 27 57 AM</td>
  </tr>
 </table>

### Changes Proposed in this PR

<table>
  <tr>
    <td>Screenshot 2020-09-10 at 5 21 10 AM</td>
<td>Screenshot 2020-09-10 at 5 29 44 AM</td>
    <td>Screenshot 2020-09-10 at 5 21 44 AM</td>
    
  </tr>
 </table>

### CODE REVIEW

**Comment by stealthanthrax** [comment]
Hey @rickhanlonii ! 👋  This is the PR for merging multi lists in a single list as discussed in the fellowship.

Please have a look.
Thanks in advance.

File: packages/rn-tester/Podfile.lock [hunk 1]
```diff
@@ -489,8 +489,8 @@ SPEC CHECKSUMS:
   CocoaAsyncSocket: 694058e7c0ed05a9e217d1b3c7ded962f4180845
   CocoaLibEvent: 2fab71b8bd46dd33ddb959f7928ec5909f838e3f
   DoubleConversion: cde416483dac037923206447da6e1454df403714
-  FBLazyVector: 8ea0285646adaf7fa725c20ed737c49ee5ea680a
-  FBReactNativeSpec: 29b1b8a11346e71351f3a2ba126439810edee362
+  FBLazyVector: 142ced51861eb7216a6fb4af384a467e0680b953
```
Comment (rickhanlonii) [inline packages/rn-tester/Podfile.lock:492]: Should the podfile have been updated?

**Review by rickhanlonii — COMMENTED** [review]
The code looks good, can you add screenshots of the before and explain what was changed for anyone reading this later and looking for context?

File: packages/rn-tester/Podfile.lock (hunk 1 above)
Comment (stealthanthrax) [reply packages/rn-tester/Podfile.lock:492]: @rickhanlonii , afair, at the time these were changed due to either `Yoga` changing at the time of development.

**Comment by stealthanthrax** [comment]
@rickhanlonii , I have updated according to the suggestions. Please let me know if anything else is required.

**Review by rickhanlonii — APPROVED** [review]

File: packages/rn-tester/Podfile.lock (hunk 1 above)
Comment (rickhanlonii) [reply packages/rn-tester/Podfile.lock:492]: The podfile didn't change so the lock file shouldn't have updated. Could you rebase on to master?

Comment (rickhanlonii) [reply packages/rn-tester/Podfile.lock:492]: Some tests are failing internally so this is my best guess for the reason.

**Comment by rickhanlonii** [comment]
@stealthanthrax can you revert the podfile change?

**Comment by stealthanthrax** [comment]
Sure @rickhanlonii !

**Comment by stealthanthrax** [comment]
@rickhanlonii , I've reverted the Podfile. Please let me know if anything else is required.

**Comment by rickhanlonii** [comment]
@sansyrox sorry for the delay, can you rebase this onto master to resolve the conflict?

**Comment by sansyrox** [comment]
Hey @rickhanlonii ,
Sure!!

**Comment by rickhanlonii** [comment]
Looks like tests are failing now

**Comment by sansyrox** [comment]
Dear "tests", why you always do this to me? :sob: 
I thought we were friends.
> Looks like tests are failing now


Jokes apart. I will try to fix them ASAP.

**Pull request closed without merging**